designs/jane/src/sleeve.mjs

```diff
@@ -0,0 +1,82 @@
+export const sleeve = {
+  name: 'jane.sleeve',
+  measurements: ['biceps', 'shoulderToElbow'],
+  options: {
+    bicepsEase: { pct: 18, min: 18, max: 51, menu: 'fit' },
+    sleeveBonus: { pct: 80, min: 60, max: 100, menu: 'style' },
+  },
+
+  draft: function draftJaneSleeve({
+    options,
+    Point,
+    Path,
+    Snippet,
+    points,
+    paths,
+    snippets,
+    sa,
+    paperless,
+    macro,
+    measurements,
+    part,
+  }) {
+    const sleeveWidth = measurements.biceps * (1 + options.bicepsEase)
+
+    const sleeveLength = measurements.shoulderToElbow * options.sleeveBonus
+
+    points.sleeveLeft = new Point(0, 0)
+    points.sleeveRight = new Point(sleeveLength, 0)
+    points.sleeveBottomLeft = new Point(0, sleeveWidth)
+    points.sleeveBottomRight = new Point(sleeveLength, sleeveWidth)
+
+    paths.sleeve = new Path()
+      .move(points.sleeveRight)
+      .line(points.sleeveLeft)
+      .line(points.sleeveBottomLeft)
+      .line(points.sleeveBottomRight)
+      .line(points.sleeveRight)
+      .close()
+
+      points.title = points.sleeveLeft.shift(300, sleeveLength / 2)
```

Comment:
Please fix the indentation, it's very misleading to structure code like this.

Reply:
Could you explain what you mean by misleading to structure code this way? I want to make sure I understand what I'm changing and why for the future.

Reply:
So this is a holdover from when complete was removed.
The current indentatinons imply that this part code should be in a bracket `{}`.
However that's not longer the case as the brackets removed when `if (complete) {}` was removed.
So now the indentation of the code that was in the complete bracket needs to be alligined with the rest of the code as is it no longer in a bracket.
It is lucky a simple fix and should it is generally fixed by `yarn prettier` when you commit new changes with git commands so I recommend doing the following.
In Gitpod or a local repo
- Remove the bracket `}` on line 163 of `body.mjs`
- Add `.addClass()` function to all paths that are not hidden. If it is meant to be just fabric then it is still needed and is `.addClass('fabric')`
- Update `.attr` to newer `.addClass()` & `.addText()` functions
- Fix the indents manually as best you can (like I said should be handled by `yarn prettier` when commiting but good to get to it manually to understand the changes)
- Possible re-name of `paths.sideSeam` to something like `paths.body`, `seam` or `box` etc,. Up to you but more than the sideSeam is included in the path so I can see why this could be confusing. 
- Re-commit the files.

Reply:
As Bob explains, indentation is used to organize code into logical blocks that belong together.
This is not required, as the computer does not care, it is for the benefit of human beings.

Loops are a classic example:

```ms
const example = 1
if (example < 0) {
    // Shift indentation to the right to indicate we're in a loop
    console.log('Inside the loop')
}
// Shift indentation back to the left to show we're out the loop
console.log('Outside the loop')
```
This makes it very easy when scanning the code to get a feel for the structure.

The same indentation is used when writing a single statement on multiple lines, like when chaining calls:

```mjs
const paths.example = new Path()
    // Shift indentation to the right to indicate we continue the statement from above
    .move(points.a)
    .line(points.b)
    .addClass('fabric')
    .addText("This is just an example)
// Shift indentation back to the left to show we're doing something new
const foo = "bar"
```

The problem with the code you wrote is that you shift the indentation to the right to indicate a chaining call (like in the second example above). But then you never shift left when it's done. And no it looks like your code is structured different than it is. You don't even have to believe me, because there was an extra closing `}` that was not supposed to be there. But of course it looked like it should have because the indentation was wrong:

```mjs
paths.extraBoundary = new Path()
    .move(points.topLeft)
    .move(points.topRight)
    .move(points.topTopRight)
    .move(points.topTopLeft)
    .move(points.topLeft)
    .close()

    points.logo = points.middle.shiftTowards(points.middleHem, maxLength / 4)
```

That line with `points.logo` on it is not shifted back to the left, which is a footgun and should be fixed.

Reply:
Thanks for explaining!